src/test/groovy/com/palantir/gradle/graal/GradleGraalEndToEndSpec.groovy

```diff
@@ -77,4 +78,103 @@ class GradleGraalEndToEndSpec extends IntegrationSpec {
         !result3.wasUpToDate(':nativeImage')
         output.getAbsolutePath().execute().text.equals("hello, world (modified)!\n")
     }
+
+    def 'allows specifying additional properties'() {
+        setup:
+        directory("src/main/java/com/palantir/test")
+        file("src/main/java/com/palantir/test/Main.java") << '''
+        package com.palantir.test;
+       
+        import java.io.IOException;
+        import java.net.URL;
+        
+        public final class Main {
+          public static final void main(String[] args) {
+            String result = null;
+            try {
+              result = convertStreamToString(new URL("http://www.google.com/").openStream());
+              System.out.println(result);
+            } catch (IOException e) {
```

Comment:
possible to declare the exception in the method signature instead to make this shorter?